### ISSUE TITLE
MNU on BlTransformationBuilder>>#inherit

### ISSUE THREAD

**Issue by tinchodias** [pharo-graphics/Bloc#183]
This method sends `transformation` to an object that belongs to `BlElementTransformation` hierarchy, but only a member understands it:

```
inherit
	"Append existing element's transformations to the new transformation list"
	
	target transformation asElementTransformation transformation
		do: [ :aTransformation | affineCompositeTransformation addTransformation: aTransformation ]
```

I found this issue several times last year but never stopped to really comprehend the idea and fix it. I'm on it now.

**Comment by tinchodias** [comment]
This error can be triggered by this example in Bloc-Pac: `BlSvgExamples>>#pathBatman`

**Comment by tinchodias** [comment]
Fixed on dev-1.0, so available for next major version.
Could be cherry-picked on master and create a hot-fix, on demand.
